SPECS/gh/gh.spec

```diff
@@ -26,10 +27,11 @@ Requires:       git
 GitHub official command line tool.
 
 %prep
-%autosetup -p1 -n cli-%{version}
```

Comment:
replaced -p1 with -N option here, so that patch will be applied to SOURCE1 as well (via autopatch below)